src/lava/magma/core/model/py/connection.py

```diff
@@ -31,8 +31,34 @@
 NUM_Y_TRACES = len(str_symbols.POST_TRACES)
 
 
-class LearningConnection:
+class AbstractLearningConnection:
     """Base class for plastic connection ProcessModels.
+    """
+
+    # Learning Ports
+    s_in_bap = None
+    s_in_y1 = None
+    s_in_y2 = None
+    s_in_y3 = None
+
+    # Learning Vars
+    x0 = None
+    tx = None
+    x1 = None
+    x2 = None
+
+    y0 = None
+    ty = None
+    y1 = None
+    y2 = None
+    y3 = None
+
+    tag_2 = None
+    tag_1 = None
+
+
+class PyLearningConnection(AbstractLearningConnection):
+    """Base class for plastic connection ProcessModels in Python / CPU.
```

Comment:
This could be "Base class for **Learning** connection ..." for consistency.  In line 64, it should be either "It has subclasses for ..." or "it is a subclass for ..".

Reply:
done